resources/models/reservation.php

```diff
@@ -221,25 +221,52 @@
             'mode' => 'time',
             'span' => 'right',
         ],
+        'registered_customer_id' => [
```

Comment:
You can use `_customer_id` here instead. Fields starting with `_` are ignored

Reply:
I don't want the field to be ignored so that I can use it conditionally on the reservation request validation to make `first_name` and `last_name` field required when `registered_customer_id` is not provided.